apps/dashboard/src/components/time-display-hover-card.tsx

```diff
@@ -0,0 +1,40 @@
+import { format, formatDistanceToNow } from 'date-fns';
+import { HoverCard, HoverCardContent, HoverCardTrigger } from './primitives/hover-card';
+
+interface TimeDisplayHoverCardProps {
+  date: Date;
+  children?: React.ReactNode;
+}
+
+export function TimeDisplayHoverCard({ date, children }: TimeDisplayHoverCardProps) {
```

Comment:
Lovely. We need this component in every timestamp we display in the Dashboard. This is already in the dashboard table for the last updated at column so let's reuse it.